nginx/prod.Dockerfile

```diff
@@ -7,6 +7,7 @@ WORKDIR /project
 COPY frontend .
 RUN yarn && yarn build
 
-FROM staticfloat/nginx-certbot
-COPY nginx/prod.nginx.conf /etc/nginx/user.conf.d/nginx.conf
+FROM jonasal/nginx-certbot:3.2.0-nginx1.23.1-alpine
+COPY nginx/redirector.conf /etc/nginx/conf.d/redirector.conf
+COPY nginx/prod.nginx.conf /etc/nginx/templates/default.conf.template
```

Comment:
> 2\. All files matching `/etc/nginx/templates/<name>.template` is run through `envsubst`

This is used in the current PR in the line above.

Reply:
Great! Then maybe this line https://github.com/MetabolicAtlas/MetabolicAtlas/blob/main/docker-compose-remote.yml#L18 is not needed anymore?

Reply:
Excellent point @nanjiangshu, that line is now removed via [2e5455f](https://github.com/MetabolicAtlas/MetabolicAtlas/pull/1158/commits/2e5455f75cc5d5dfaa02db26bc8a6dab5eaf66b1).